src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -859,6 +869,19 @@ impl VirtioDevice for Net {
             }
         }
 
+        let desired_flags = [
+            (gen::TUN_F_CSUM, VIRTIO_NET_F_CSUM),
+            (gen::TUN_F_UFO, VIRTIO_NET_F_GUEST_UFO),
+            (gen::TUN_F_TSO4, VIRTIO_NET_F_GUEST_TSO4),
+            (gen::TUN_F_TSO6, VIRTIO_NET_F_GUEST_TSO6),
+        ];
```

Comment:
I guess because this 'map' is used here and in all tests, let's make it a `const` at the top of the file.

Reply:
but it's different in some tests, isn't it?

Reply:
In that test you can take a slice (because it only lacks last element): `let desired_flags = FLAGS_MAP[..FLAGS_MAP.len() - 1]` where `FLAGS_MAP` is the name of the constant.